package.json

```diff
@@ -39,12 +39,12 @@
     "@tanstack/react-query": "^4.36.1",
     "@tanstack/react-query-persist-client": "^4.36.1",
     "@tanstack/react-table": "^8.16.0",
-    "@thirdweb-dev/chains": "0.1.106-nightly-4de88024357cf8197ec78fb4fabaf5bddd47c605-20240509062610",
-    "@thirdweb-dev/react": "4.6.16-nightly-4de88024357cf8197ec78fb4fabaf5bddd47c605-20240509062610",
-    "@thirdweb-dev/sdk": "4.0.77-nightly-4de88024357cf8197ec78fb4fabaf5bddd47c605-20240509062610",
+    "@thirdweb-dev/chains": "0.1.109-nightly-6d36a1498a533c6184c90fff65699465f9786b2f-20240513223521",
+    "@thirdweb-dev/react": "4.6.21-nightly-6d36a1498a533c6184c90fff65699465f9786b2f-20240513223521",
+    "@thirdweb-dev/sdk": "4.0.82-nightly-6d36a1498a533c6184c90fff65699465f9786b2f-20240513223521",
     "@thirdweb-dev/service-utils": "0.4.28",
     "@thirdweb-dev/storage": "2.0.14",
```

Comment:
need to update these, too

Reply:
these weren't built in the nightly build (https://github.com/thirdweb-dev/js/actions/runs/9070684962/job/24922947662)

Reply:
updated